pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -378,10 +378,10 @@ func (m *manager) removeStaleState() {
 	// Loop through the CPUManager state. Remove any state for containers not
 	// in the `activeContainers` list built above.
 	assignments := m.state.GetCPUAssignments()
-	for podUID := range assignments {
-		for containerName := range assignments[podUID] {
+	for podUID, containers := range assignments {
+		for containerName := range containers {
```

Comment:
what's the benefit of this specific change?

Reply:
No benefit performance-wise, this seems more idiomatic Go to me :) And one less map lookup